plugin.cpp

```diff
@@ -174,11 +201,33 @@ void plugin_ingest(PLUGIN_HANDLE *handle,
 				continue;
 			}
 		}
+		else
+		{
+			Logger::getLogger()->warn("No asset name match configured");
+		}
+
+		if (tracker)
+		{
+			tracker->addAssetTrackingTuple(info->configCatName, (*elem)->getAssetName(), string("Filter"));
+		}
 		// Get a reading DataPoint
 		const vector<Datapoint *>& dataPoints = (*elem)->getReadingData();
 		// Iterate over the datapoints
 		for (vector<Datapoint *>::const_iterator it = dataPoints.begin(); it != dataPoints.end(); ++it)
 		{
+			if (!datapoint_match.empty())
+			{
+				string datapoint_name = (*it)->getName();
+				if (! regex_match(datapoint_name, *dp_re))
+				{
+					continue;
+				}
+			}
+			else
+			{
+				Logger::getLogger()->warn("No datapoint match configured");
```

Comment:
See my comment above. Lack of datapoint filter string should not trigger a warning either.

Reply:
Changed to debug level